nslsii/__init__.py

```diff
@@ -320,38 +332,45 @@ def configure_ipython_exc_logging(exception_logger, ipython, rotate_file_size=10
         at the time configure_ipython_exc_logging() is called, if there exists a log file
         with size in bytes greater than or equal to rotate_file_size, the existing file will
         be renamed and a new log file will be created
+    appdirs_appname: str
+        appname passed to appdirs.user_log_dir(); use the default for production,
+        set to something else for testing
 
     Returns
     -------
-    bluesky_ipython_log_file_path: str
+    bluesky_ipython_log_file_path: Path
         log file path
 
     """
     ipython.set_custom_exc((BaseException,), exception_logger)
 
     if "BLUESKY_IPYTHON_LOG_FILE" in os.environ:
+        bluesky_ipython_log_file_path = Path(os.environ["BLUESKY_IPYTHON_LOG_FILE"])
```

Comment:
Same question as above for existence checking.

Reply:
see answer above